README.md

```diff
@@ -1 +1,13 @@
-# tictactoe-java
+# tictactoe-java [![Build Status](https://travis-ci.org/klkelley/tictactoe-java.svg?branch=skeleton-setup)](https://travis-ci.org/klkelley/tictactoe-java)
+
+
+### Build & Run
+```
+$ mvn clean install
+$ mvn exec:java
```

Comment:
I'd like to be able to run the project without `mvn`. So look into how to build a jar file, and give me instructions on how to do that (using `mvn`) and then run that jar file. Once that's done I'm cool with this being merged.